va/va.go

```diff
@@ -87,13 +87,17 @@ func certNames(cert *x509.Certificate) string {
 		names = append(names, cert.Subject.CommonName)
 	}
 	names = append(names, cert.DNSNames...)
+	for i := range cert.IPAddresses {
```

Comment:
I don't think its worthwhile updating `certNames` to also include IP addresses in the joined names, it's only used by `validateTLSALPN01` and only when presented with an incorrect validation certificate. Can you back this diff out?